src/MSALWrapper.Test/AuthFlow/BrokerTest.cs

```diff
@@ -44,39 +46,19 @@ public class BrokerTest
         [SetUp]
         public void Setup()
         {
-            // Setup in memory logging target with NLog - allows making assertions against what has been logged.
-            var loggingConfig = new NLog.Config.LoggingConfiguration();
-            this.logTarget = new MemoryTarget("memory_target");
-            loggingConfig.AddTarget(this.logTarget);
-            loggingConfig.AddRuleForAllLevels(this.logTarget);
+            (this.logger, this.logTarget) = MemoryLogger.Create();

```

Comment:
we are updating each test with this, which is great! We simplified it by moving memory log setup into it's own thing. However, none of these tests are asserting the log messages though. A lot of tests do have logs and we should probably assert those.

Reply:
Correct, I've intentionally avoided changing the _behavior_ of any tests to keep this PR scoped to a `refactor`, and not an `enhancement` of the tests themselves. We could have made the same enhancements already, this is more about the deduplication of the logging setup.